aws/scripts/README.md

```diff
@@ -0,0 +1,40 @@
+# AWS Multi Account Onboarding Instructions
+
```

Comment:
Since you are going this far, it would be great to share a little blurb about why you might want to use this script.

Also, it seems a bit unfindable by putting it in scripts/.

We can also put these docs in KB instead if you prefer, and leave the OSS repo less documented, if that's easier.

If I zoom out, this script is valuable in a generic fashion for any AWS user who wants to enable organizational monitoring. I would think of sharing this with the world in that regard, and documenting the scripts appropriately. It could be its own OSS repo that people not using Ternary would still benefit from.

Reply:
Gotcha- that works for me. I have documented the same onboarding steps the script does except via the UI [here](https://coda.io/d/Ternary-Wiki_d-dBjbKJHMI/Customer-AWS-Multi-Account-Monitoring-Setup_sulwl#_luXcd), and I was going to link the script in there. Happy to move this code or have it live wherever you think is best though

Reply:
Let's just keep it in the back of our head, I will bring this up at TT when Bruce returns.